packages/react-scripts/template/README.md

```diff
@@ -1833,17 +1832,6 @@ following into account:
    instructions for using other methods. _Be sure to always use an
    incognito window to avoid complications with your browser cache._
 
-1. If possible, configure your production environment to serve the generated
```

Comment:
As per https://developers.google.com/web/updates/2018/06/fresher-sw, I think we can just leave this section out now—I don't know how much the README needs to cater to older browser?

Reply:
Since service workers are only supported in evergreen browsers, I'm fine with this.